dev/io.openliberty.microprofile.telemetry.1.1.internal/src/io/openliberty/microprofile/telemetry11/internal/rest/jakarta/TelemetryClientBuilderListener.java

```diff
@@ -16,7 +16,9 @@
 import com.ibm.websphere.ras.Tr;
 
 import io.openliberty.restfulWS.client.ClientBuilderListener;
-import jakarta.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.ClientBuilder;
+import io.openliberty.microprofile.telemetry11.internal.rest.TelemetryClientFilter;
+import com.ibm.ws.jaxrs20.providers.api.JaxRsProviderRegister;
```

Comment:
Import added but not referenced anywhere which again suggests formatting hooks are not running.